presto-docs/src/main/sphinx/admin/tuning.rst

```diff
@@ -10,29 +10,392 @@ Config Properties
++                                          +                                          +                                    +scheduling splits. ``flat`` will try to schedule       +
++                                          +                                          +                                    +splits on the same host as the data is located by      +
++                                          +                                          +                                    +reserving 50% of the work queue for local splits.      +
++------------------------------------------+------------------------------------------+------------------------------------+-------------------------------------------------------+
++       ``task.default-concurrency``       +               ``Integer``                +               ``1``                +Default local concurrency for parallel operators. Use  +
++                                          +                                          +                                    +this as default for sessions properties:               +
++                                          +                                          +                                    +``task_hash_build_concurrency`` and                    +
++                                          +                                          +                                    +``task_aggregation_concurrency`` as well as            +
++                                          +                                          +                                    +``task.join-concurrency`` property.                    +
++------------------------------------------+------------------------------------------+------------------------------------+-------------------------------------------------------+
++        ``task.join-concurrency``         +               ``Integer``                +``task.default-concurrency`` (``1``)+Servers as default value for session property:         +
++                                          +                                          +                                    +``task_join_concurrency``. Describes local concurrency +
++                                          +                                          +                                    +for join operators. This value may be increased to     +
++                                          +                                          +                                    +perform join on worker using more then one thread.     +
++                                          +                                          +                                    +This will increase CPU utilization with the cost of    +
++                                          +                                          +                                    +increased memory usage.                                +
```

Comment:
I agree it's not obvious that memory usage will increase significantly (it quite easy to show that it will increase at least slightly though). I would add here information about context switching cost if this value is to high.

As for the wiki page, I think in this particular case it's not required, but in general I maybe we can create Jira task for properties we want to investigate further as it will generate additional workload?